Module-2 | Day-5/clientCreate.js

```diff
@@ -0,0 +1,35 @@
+var Request = require("request");
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+app.get('/:companyname/:country/:timezone', (req,res) => {
+
+   
+
+   const companyname = req.params.companyname;
+   const country = req.params.country;
+   const timezone = req.params.timezone;
+    
+   Request.post({
+    "headers": { "content-type": "application/json", "Authorization":"Basic YWIzMDczNDYxMTY5YmU0ODJjNGJlYzY0OGMzYzJlMjc6" },
```

Comment:
why is Authorization token hardcoded? This system would only work for campaign monitor account this way. You should take the token as an iniput in your API call and then call the api accordingly.